api/bugzilla/api_bugzilla.py

```diff
@@ -80,8 +145,141 @@ def __init__(self):
         self.BugzillaHelperClient = BugzillaHelper()
 
     def contains_flags(self, entry, criteria):
+        print("CONTAIN_FLAGS")
         return all(entry.get(key) == value for key, value in criteria.items())
 
+    def get_fixed_bug_ids(self) -> list[int]:
+        # Get bugs with resolution is FIXED from DB
+        R = ReportBugzillaSoftvisionBugs
+        norm_res = func.upper(func.trim(R.bugzilla_bug_resolution))
+
+        q = (
+            self.db.session
+            .query(distinct(R.bugzilla_key))
+            .filter(norm_res == 'FIXED')               # robust string compare
```

Comment:
Do we care about `WONTFIX` or `INVALID`? Sometimes bugs marked “WONTFIX” or “INVALID” might still have release flags set. If those matter, they’ll be skipped. Unless that's not a thing and they remove flags on those. Can you look up any examples from Bugzilla with this?

Also reopened bugs, if a bug was once fixed, then reopened, `get_fixed_bug_ids` might still include it, even though it’s no longer fixed.